index.html

```diff
@@ -59,134 +72,148 @@ <h2 class="connect-me">LET'S CONNECT</h2>
-            <span class="role-and-year">Back End Dev</span>
-            <span class="role-and-year">2015</span>
-          </div>
+                <div class="stats">
+                    <span class="company">CANOPY</span>
+                    <span class="role-and-year">Back End Dev</span>
+                    <span class="role-and-year">2015</span>
+                </div>
 
-          <p class="description">
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-          </p>
+                <p class="description">
+                    A daily selection of privately personalized reads; no accounts or
+                    sign-ups required.
+                </p>
 
-          <ul class="languages">
-            <li class="language">html</li>
-            <li class="language">css</li>
-            <li class="language">javascript</li>
-          </ul>
+                <ul class="languages">
+                    <li class="language">html</li>
+                    <li class="language">css</li>
+                    <li class="language">javascript</li>
+                </ul>
 
-          <button class="btn" type="button">See Project</button>
+                <button class="btn" type="button">See Project</button>
+          </div>
+          
         </li>
 
         <li class="project">
-          <img src="./img/project2.png" alt="" />
-          <h2 class="project-title">Multi-Post Stories</h2>
-          <div class="stats">
-            <span class="company">CANOPY</span>
-            <span class="role-and-year">Back End Dev</span>
-            <span class="role-and-year">2015</span>
-          </div>
-          <p class="description">
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-          </p>
+          <img src="./img/project4.png" alt="" />
+          <div class="project-info">
+                <h2 class="project-title">Multi-Post Stories</h2>
+                <div class="stats">
+                    <span class="company">FACEEBOOK</span>
+                    <span class="role-and-year">Full Stack Dev</span>
+                    <span class="role-and-year">2015</span>
+                </div>
+                <p class="description">
+                    Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.
+                </p>
 
-          <ul class="languages">
-            <li class="language">html</li>
-            <li class="language">css</li>
-            <li class="language">javascript</li>
-          </ul>
+                <ul class="languages">
+                    <li class="language">html</li>
+                    <li class="language">Ruby on Rails</li>
+                    <li class="language">css</li>
+                    <li class="language">javascript</li>
+                </ul>
```

Comment:
- [ ] Kindly adjust the image and project-info to fit in the required change. 
- The image should occupy half of the card, increase the height and width.
- The project-info should be properly aligned horizontally. The `role-and-year` style is also affecting the required design for that section.
- Make the same changes for the remaining projects.
_**Refer to my comments above about the design.**_ 
_See the images below for clarity._

**Original** | **Your clone**
  ----------- | -----------
  ![image](https://user-images.githubusercontent.com/49621279/121508124-218d7f80-c99a-11eb-9208-e3d7c623f36c.png) | ![image](https://user-images.githubusercontent.com/49621279/121508186-32d68c00-c99a-11eb-9723-8b7fa910559e.png)